recipes/highs/all/conanfile.py

```diff
@@ -27,10 +24,6 @@ class HiGHSConan(ConanFile):
         "fPIC": True,
     }
 
-    def validate(self):
```

Comment:
So I take that there was no issue with it and ir always worked? Oh

Reply:
I forgot to copy the .lib file on Windows in the initial version of the recipe. I didn't know how to fix the problem of missing symbols. My only solution was to deactivate shared libs on Windows. I know better now.